src/shared_modules/dbsync/include/dbsync.hpp

```diff
@@ -209,9 +209,10 @@ class EXPORTED DBSyncTxn
      * @brief Gets the deleted rows (diff) from the database.
      *
      * @param callbackData    Result callback(std::function) will be called for each result.
+     * @param options
      *
      */
-    virtual void getDeletedRows(ResultCallbackData callbackData);
+    virtual void getDeletedRows(ResultCallbackData callbackData, const nlohmann::json& options);
```

Comment:
You might have thought of this already, but maybe we could add a new overload that takes an `options` field?

Reply:
Nice idea! This was applied as DoD request, but what do you think about set a default value?

Reply:
I think a default value would be just a good 👍